gpib_server.py

```diff
@@ -176,10 +189,26 @@ def read(self, c, n_bytes=None):
         Otherwise, reads until the device stops sending.
         """
         instr = self.getDevice(c)
-        if n_bytes is None:
-            ans = instr.read_raw()
+        if instr.resource_name.startswith('USB'):
+            try:
+                if n_bytes is None:
+                    ans = instr.read()
+                else:
+                    ans = instr.read(n_bytes)
+            except:
+                self.asynchronous_wait(0.1)
```

Comment:
this would need a `yield` to actually wait for the deferred to fire before continuing:
```python
yield self.asynchronous_wait(0.1)
```
or
```python
yield labrad.util.wakeupCall(0.1)
```